cmd/gitops/root/cmd.go

```diff
@@ -85,6 +88,31 @@ func RootCmd() *cobra.Command {
 				fmt.Fprintf(os.Stderr, "Error: %v\n", err)
 				os.Exit(1)
 			}
+
+			_, err = config.GetConfig(false)
+			if err != nil {
+
+				prompt := promptui.Prompt{
+					Label:     "Creating config file...would you like to turn on analytics to help us improve our product",
+					IsConfirm: true,
+					Default:   "Y",
+				}
+
+				// Answering "n" causes err to not be nil. Hitting enter without typing
+				// does not return the default.
+				_, err := prompt.Run()
+				gitopsConfig := &config.GitopsCLIConfig{}
+				seed := time.Now().UnixNano()
+				gitopsConfig.UserID = config.GenerateUserID(10, seed)
+				gitopsConfig.Analytics = false
```

Comment:
❔ Should not the value of `Analytics` be set by default to false in a new config? @ozamosi WDYT, is this assignment to false needed here? Can we keep just:

```
if err == nil {
    gitopsConfig.Analytics = true
}
```